src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -127,31 +128,31 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
     'classifiedDomains',
   ]);
 
-  const {videoInput: currentCameraDevice} = useKoSubscribableChildren(mediaDevicesHandler.currentDeviceId, [
-    DeviceTypes.VIDEO_INPUT,
+  const {videoinput: currentCameraDevice} = useKoSubscribableChildren(mediaDevicesHandler.currentDeviceId, [
```

Comment:
This doesn't work ? 🤔 

```suggestion
  const {[MediaDeviceType.VIDEO_INPUT]: currentCameraDevice} = useKoSubscribableChildren(mediaDevicesHandler.currentDeviceId, [
```